_episodes/09-pull-requests.md

```diff
@@ -192,99 +305,237 @@ Largest City: Paris
+  (use "git add <file>..." to mark resolution)
+
+	both modified:   README.md
+
+no changes added to commit (use "git add" and/or "git commit -a")
+~~~
+{: .output}
+
+Now we will resolve the conflict by editing the `README.md` file to contain both 
+'United States' and 'France' and none of the additional lines git added to
+indicate conflict
+
+~~~
+$ nano README.md
+$ cat README.md
 ~~~
 {: .bash}
 
-Now if we reload the pull request.  The new commit was added to that pull request.
+~~~
+# countries
+Sandbox for ComBEE github workshop on PR's
 
-![](../figgithub_screenshot_after_new_commit.png)
+Countries:
+France
+United States
+~~~
+{: .output}
 
-This is great for our purposes but you can imagine you may want to suggest changes that
-aren't linked to one another.
-This is one place where branches come in.
-We can put the changes in a different branch and make a separate pull request for that branch.
-Branches can also be useful when you are developing a new function in your script and you
-don't want to 'break' the current version that is published in your repo.
+Then we add and commit our resolved conflict.
 
-We will use a new branch and pull request to suggest the addition of a second country.
-You can create a branch and then switch to it with the following commands:
 
 ~~~
-$ git branch addItaly
-$ git checkout addItaly
+$ git add README.md
+$ git commit -m "Resolved conflict in readme w two countries"
 ~~~
 {: .bash}
 
 ~~~
-Switched to branch 'addItaly'
+[addFrance 912317b] Resolved conflict in readme w two countries
 ~~~
 {: .output}
 
-**OR** you can create the branch and switch to it in one step
+Finally we can update the pull request by pushing these changes to our github 
+fork of the repository.
 
 ~~~
-$ git checkout -b addItaly
+$ git push origin addFrance
 ~~~
 {: .bash}
 
 ~~~
-Switched to branch 'addItaly'
+git push origin addFrance
+Counting objects: 6, done.
+Delta compression using up to 4 threads.
+Compressing objects: 100% (6/6), done.
+Writing objects: 100% (6/6), 732 bytes | 0 bytes/s, done.
+Total 6 (delta 2), reused 0 (delta 0)
+remote: Resolving deltas: 100% (2/2), completed with 1 local object.
+To https://github.com/sstevens2/countries.git
+   66d7ebf..912317b  addFrance -> addFrance
 ~~~
 {: .output}
 
-> ## Add new country file and make PR
+Now if we reload our browser we will see that the new commit is in the pull 
+request and it has no conflicts with the base branch.
+
+![](../fig/github_screenshot_resolved_PR_conflict.png)
+
+Now the owner/administrator/manager of the authoritative repo can review our pull requests and decide to incorporate them.
+
+
+> ## Add new country file and make additional PR
 >
-> - Copy your first country file into a new country
+> - Starting in the master branch make a new branch
+> - Copy other country file into a new country
 > - Edit the file to include info on the new country
 > - Add and commit this new file
 > - Push the new changes to github
 > > ## Solution
```

Comment:
This exercise isn't rendering :cry: 

I *think* there is an empty line (only a `>`) missing before the solution section.